src/sentry/static/sentry/app/views/alerts/wizard/index.tsx

```diff
@@ -3,13 +3,18 @@ import {RouteComponentProps} from 'react-router';
 import styled from '@emotion/styled';
 
 import Feature from 'app/components/acl/feature';
+import CreateAlertButton from 'app/components/createAlertButton';
```

Comment:
do we have to use this

Reply:
i was thinking it could be useful because we could leverage some of the `CreateAlertFromViewButton` functionality since the first release of the wizard could be accomplished by having an associated eventview with each alert wizard template.

Reply:
If you'd rather us not use this component though I could look into changing this to basically just be a link and generate the query params manually

Reply:
its fine, just have to make sure it isn't possible to display an error since they can't change anything from this page

Reply:
gotcha I'll keep that in mind